torch/_inductor/cpp_builder.py

```diff
@@ -1417,11 +1417,11 @@ def __init__(
             sources = [sources]
 
         if config.is_fbcode():
-            if self._aot_mode and not self._use_absolute_path:
+            if self._aot_mode and not self._use_relative_path:
                 inp_name = sources
-                # output process @ get_name_and_dir_from_output_file_path
             else:
-                # We need to copy any absolute-path torch includes
+                # Will create another temp director for building, so do use
```

Comment:
Using the absolute path should be in the `not self._use_relative_path` branch if we use the new naming?

Reply:
I missed a "NOT".. I will add it in another follow-up PR to avoid another long CI run.